README.md

```diff
@@ -39,10 +39,16 @@ If you installed Docker for Mac, `docker-compose` is also installed. Otherwise i
 
 Note: If installing via pip fails due to pip failing to upgrade your system Python packages, try running `pip install hokusai --ignore-installed`.
 
-Alternatively, you can workaround a pure Python installation and install a binary for OSX by simply downloading it from https://artsy-provisioning-public.s3.amazonaws.com/hokusai and adding it to your `$PATH`, i.e.
+Alternatively, you can workaround a pure Python installation and install a binary for OSX by simply downloading it from https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-{version}-{platform}-x86_64 and adding it to your `$PATH`, i.e.
 
+MacOS:
 ```
-curl https://artsy-provisioning-public.s3.amazonaws.com/hokusai -o /usr/local/bin/hokusai && chmod +x /usr/local/bin/hokusai
+curl https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-latest-Darwin-x86_64 -o /usr/local/bin/hokusai && chmod +x /usr/local/bin/hokusai
+```
+
+Linux:
+```
+curl https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-latest-Linux-x86_64 -o /usr/local/bin/hokusai && chmod +x /usr/local/bin/hokusai
```

Comment:
What do you think of documenting a curl invocation supporting both platforms with `unname`?

```
$ curl --silent https://artsy-provisioning-public.s3.amazonaws.com/hokusai/hokusai-latest-$(uname -s)-$(uname -m) -o /usr/local/bin/hokusai && chmod +x /use/local/bin/hokusai
```

Reply:
👍